api/db-cdr/changelog-schema/db.changelog-8-achilles-indexes.xml

```diff
@@ -0,0 +1,93 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<databaseChangeLog
+        xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+        xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+        xmlns:ext="http://www.liquibase.org/xml/ns/dbchangelog-ext"
+        xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.1.xsd
+        http://www.liquibase.org/xml/ns/dbchangelog-ext http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-ext.xsd">
+  <changeSet author="biopete" id="changelog-8 add concept indexes ">
+    <validCheckSum>ANY</validCheckSum>
+
+    <createIndex
+                 indexName="idx_concept_name"
+                 schemaName="cdr"
+                 tableName="concept"
+                 unique="false">
+      <column name="concept_name" type="varchar(255)"/>
+    </createIndex>
+    <createIndex
+            indexName="idx_concept_code"
+            schemaName="cdr"
+            tableName="concept"
+            unique="false">
+      <column name="concept_code" type="varchar(50)"/>
+    </createIndex>
+    <createIndex
+            indexName="idx_domain_id"
+            schemaName="cdr"
+            tableName="concept"
+            unique="false">
+      <column name="domain_id" type="varchar(20)"/>
+    </createIndex>
+
+    <createIndex
+            indexName="idx_vocabulary_id"
+            schemaName="cdr"
+            tableName="concept"
+            unique="false">
+      <column name="vocabulary_id" type="varchar(20)"/>
+    </createIndex>
+    <createIndex
+            indexName="idx_standard_concept"
+            schemaName="cdr"
+            tableName="concept"
+            unique="false">
+      <column name="standard_concept" type="varchar(1)"/>
+    </createIndex>
+
+
+  </changeSet>
+
+  <changeSet author="biopete" id="changelog-8.1 add achilles_results_concept indexes ">
```

Comment:
Any specific reason why this is in a different changset than the things above it? Seems like everything in the file could be in the same changeset?

Reply:
making different changesets helped with development i think. Bceause if i had an error in one changeset, it did not try to run the ones that had already run. So making a bunch of little change sets makes development easier.  I'm 99 percent sure that is why i did that.